cabot/cabotapp/models.py

```diff
@@ -111,7 +111,6 @@ class Meta:
     schedules = models.ManyToManyField(
         'Schedule',
         blank=True,
-        null=True,
```

Comment:
Yup - found because Django added a cool new "system check" framework that lints things like these (which iirc is run automatically when you do migrations/start a dev server).